manifests/install/repos/yum.pp

```diff
@@ -1,8 +1,13 @@
 # Install a yum repo
 define foreman::install::repos::yum ($repo, $yumcode, $gpgcheck) {
   $repo_path = $repo ? {
-    'stable' => 'releases/latest',
-    default  => $repo,
+    'stable'  => 'releases/latest',
+    'nightly' => $repo,
+    default   => "releases/${repo}",
```

Comment:
We might need to handle the case where somebody's already passing "releases/1.7" or similar in, so we don't prepend "releases/" again.